### PR TITLE
Added a pipeline to run linting checks on pull requests

### PR DESCRIPTION
**Pull request by FGRCL**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,15 +3,19 @@
 buildscript {
     ext.kotlin_version = '1.3.50'
     ext.objectboxVersion = '2.5.1'
+    ext.ktlint_version = '9.2.1'
```
Comment (felixlapierre) [inline build.gradle:6]: Can this be put in the app/build.gradle or does it have to be here?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: android
+dist: trusty
```
Comment (felixlapierre) [inline .travis.yml:2]: What's trusty?

**Review by felixlapierre — APPROVED** [review]
Looks good!

File: build.gradle (hunk 1 above)
Comment (FGRCL) [reply build.gradle:6]: I think it has to be there, since the plugin is installed project-wide

File: .travis.yml (hunk 2 above)
Comment (FGRCL) [reply .travis.yml:2]: The default android travis-ci file includes this. It seems to specify the build environment. Here's the documentation page on it.
https://docs.travis-ci.com/user/reference/trusty/

**Pull request merged**